scripts/docker-test-wrapper-zsh.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Mocking MacOS programs for linux based docker container 
+
+mkdir -p /tmp/test-metadata
+
+export TMPDIR="/tmp"
+export INSTALLER_EXTRA_ARGS="linux --init none"
+export SHELL="zsh"
```

Comment:
this should be the full path to the shell binary `/bin/zsh`

Reply:
Also could you please try setting the default shell of the root user to zsh in the dockerfile so that we don't have to set this manually? The benefit being it would make the test closer to a real setup

Reply:
Attempted to install zsh and use `chsh` to have the appropriate environment, however this does not play nicely with docker.